store/boardSlice.js

```diff
@@ -21,19 +21,30 @@ export const boardSlice = createSlice({
           action.payload.stack.cards[card.id] = card
         })
       }
+      // Filter out existing stack with same id
```

Comment:
I made sense of all your PR except for the reason of this test.

In what case(s) do you think a stack with the same id could exist in the store?

Reply:
Fair question 🙂 

It looks like whenever `BoardDetails` mounts, you send off an API request and save the result to the store. Previously duplicates would not exist, because you would simply replace the stack at the index of the stack's id. With the changed behavior, however, whenever the API returns with the stacks, they would be pushed on top of all existing ones. In this method: https://github.com/StCyr/deck-react-native/blob/ed2bfd63b834ceb1ee99045d155aa470ab2a6aa4/components/BoardDetails.js#L213

Hope this answers your question 🙂